### ISSUE TITLE
IDs in the Implementation documentation are not unique and can be misleading/confusing

### ISSUE THREAD

**Issue by hsilva-keep** [DILCISBoard/E-ARK-CSIP#124]
IDs in the Implementation documentation (https://dilcisboard.github.io/E-ARK-CSIP/implementation/) are not unique and can be misleading/confusing. 

I was creating some issues on GitHub, and then I realized that the IDs I was mentioning were not unique (e.g. by doing a search in the text).

Using **CSIP5** as example, it appear in two different sections for different purposes:
- **CSIP5**: The Information Package folder MUST include a folder named metadata, which MUST include at least all metadata relevant for the whole package
- **CSIP5** Other Content Information Type Specification

Bottom line, I think IDs and any other way one might use to reference to a certain piece of the text should be unique.

**Comment by karinbredenberg** [comment]
The ID's havent been set in their final version since we are working with them currently and wont be ready until the text are ready. The rule we have decided upon is that the requirement are named with an abbreviation and a running number. The running number cant be reused if a requirement is removed which means during this development phase we will have them have different numbers.

**Comment by kuldaraas** [comment]
Updated with the latest pull request
